src/apps/common/submodules/portListing/portListing.js

```diff
@@ -100,15 +100,21 @@ define(function(require) {
 						}
 					};
 				}),
-				tabs = _.map(self.appFlags.portListing.tabs, function(tab) {
-					return {
-						text: self.i18n.active().portListing.tabs[tab],
-						id: tab,
-						menus: [{
-							tabs: subTabs
-						}]
-					};
-				}),
+				tabs = _
+					.chain(self.appFlags.portListing.tabs)
+					.filter(function(tab) {
```

Comment:
This could be switched to use `lodash#reject` for clarity, especially when ready the condition.

Reply:
Sounds good, the intent will be more clear with `lodash#reject`.